docs/install/index.rst

```diff
@@ -0,0 +1,23 @@
+.. _install-index:
+
+##################
+Installation Guide
+##################
+
+
+This section provides instructions to install Rubix and use it with Presto or
+Hive. Instructions are tested on an EMR 5.X cluster. Same instructions should work
+with installations on-prem and other distributions. Please `contact <https://groups.google.com/forum/#!forum/rubix-users>`_
```

Comment:
what do you mean by "other distributions" here?  other distribution of Presto /Hive version? 
What are the versions we tested in this guide?

Reply:
I have clarified that it is tested on EMR 5.x only.